Bottom: 8edf79c83b21dbdf9b1c6049919c5f55cc95c6b5
Top:    5f84861cc3f7b146da7e8494a2f876401fd91bad
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-04-21 02:08:37 +0100

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
